examples/06-csi-drivers.yaml

```
@@ -0,0 +1,16 @@
+# An example of ClusterConfig object with access to CSI drivers:
+---
+apiVersion: eksctl.io/v1alpha4
+kind: ClusterConfig
+
+metadata:
+  name: cluster-1
```

Let's match the current convention:
```suggestion
  name: cluster-6
```